jenkins/ci.suse.de/openstack-ardana-pcloud.yaml

```diff
@@ -51,72 +51,109 @@
       - choice:
           name: scenario_name
           choices:
-            - entry-scale-kvm
-            - entry-scale-swift
```

Comment:
entry-scale-swift is a valid example scenario.. Why are we removing the support for this scenario? Is it because we can accomplish this same scenario by choosing the entry-scale-kvm but with relevant services disabled?

Reply:
> entry-scale-swift is a valid example scenario.. Why are we removing the support for this scenario? Is it because we can accomplish this same scenario by choosing the entry-scale-kvm but with relevant services disabled?

@mbelur I agree that `entry-scale-swift` is a valid example input model, but the input model scenario name actually points to one of the available files [here](https://github.com/SUSE-Cloud/automation/tree/master/scripts/jenkins/ardana/ansible/roles/input_model_generator/vars) and we don't have an input model generator scenario file defined for `entry-scale-swift` yet.
The input model generator is thoroughly documented [here](https://github.com/SUSE-Cloud/automation/blob/master/scripts/jenkins/ardana/ansible/roles/input_model_generator/README.md), if you want to take a deeper dive into its inner workings.